c/aws-c-common/aws_priority_queue_init_dynamic_harness-1.i

```diff
@@ -7911,8 +7911,8 @@ void aws_priority_queue_init_dynamic_harness() {
 
 
     struct aws_allocator *allocator = can_fail_allocator();
-    size_t item_size;
-    size_t initial_item_allocation;
+    size_t item_size = nondet_size_t();
+    size_t initial_item_allocation = nondet_size_t();
     size_t len;
```

Comment:
Shouldn't `len` also be initialised in the same way?

Reply:
`len` is initialized a few lines later by a call to `aws_mul_size_checked` :https://github.com/sosy-lab/sv-benchmarks/blob/master/c/aws-c-common/aws_priority_queue_init_dynamic_harness-1.i#L7921